nextflow.config

```diff
@@ -229,46 +234,68 @@ profiles {
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
+        apptainer.enabled      = false
     }
     docker {
         docker.enabled         = true
+        //docker.registry        = 'quay.io'
```

Comment:
You deactivate it? Can you comment on why exactly?

Reply:
Ah probably because the modules without quay.io as default registry (like gunzip and untar) have not been updated yet...

Reply:
yep! Easy to reactivate in future